build/configs/artik05x/artik05x_download.sh

```diff
@@ -240,7 +256,7 @@ while test $# -gt 0; do
 		--verify)
 			VERIFY=verify
 			;;
-		ALL|OS|APPS|ROM|BL1|BL2|SSSFW|WLANFW|OTA|SYSTEM|all|os|apps|rom|bl1|bl2|sssfw|wlanfw|ota|system)
+		ALL|OS|APPS|ROM|BL1|BL2|SSSFW|WLANFW|OTA|SYSTEM|MICOM|WIFI|all|os|apps|rom|bl1|bl2|sssfw|wlanfw|ota|micom|wifi)
```

Comment:
You added `micom|wifi` with `system` removed. But You added `MICOM|WIFI` without `SYSTEM` removed. What is right?